dll/cpl/desk/lang/tr-TR.rc

```diff
@@ -28,7 +35,7 @@ BEGIN
     PUSHBUTTON "&Renk...", IDC_COLOR_BUTTON, 188, 167, 55, 15
     LTEXT "&Yerleşim:", IDC_STATIC, 188, 128, 55, 9
     COMBOBOX IDC_PLACEMENT_COMBO, 188, 139, 55, 54, CBS_DROPDOWNLIST | WS_VSCROLL | WS_TABSTOP
-    PUSHBUTTON "Customize &Desktop...", IDC_DESKTOP_CUSTOM, 8, 194, 90, 15
+    PUSHBUTTON "Masaüstünü &Kişiselleştir", IDC_DESKTOP_CUSTOM, 8, 194, 90, 15
```

Comment:
Hold that PR back for a while until my desk.cpl PR was merged. Then check again that your &k here will not collide with the &k that will be placed at line 33. You are free to change either line38 or line 33 then.

Reply:
sure

Reply:
Okay, I do think you now better my comment from 2 days ago. Check which accels 2k3sp2 turkish do use for both lines. Currently they do collide after your current change, as I anticipated. Adding a picture of the MS 2k3sp2 turkish dialog would be desreable, so that we can match the accels that MS uses.